tarxx-example.cpp

```diff
@@ -26,13 +26,43 @@
 
 #include "tarxx.h"
 
-int main(const int argc, const char* const* const argv)
-{
+int tarFile(const int argc, const char* const* const argv) {
     if (argc < 2) return 1;
-    tarxx::tarfile tar(argv[1]);
+    tarxx::tarfile tar(argv[1] + std::string("_file.tar"));
     if (!tar.is_open()) return 2;
     for (unsigned i = 2; i < argc; ++i) {
         tar.add_file(argv[i]);
     }
     return 0;
 }
+
+int tarStreaming(const int argc, const char* const* const argv) {
```

Comment:
suggest calling this `main_streaming` or `main_tar_streaming` or `main_streaming_mode`; at least for style consistency `tar_streaming`.

Reply:
it's `tar_file_mode` now